frontend/packages/pipelines-plugin/src/utils/pipeline-augment.ts

```diff
@@ -210,9 +210,18 @@ export const getPipelineFromPipelineRun = (pipelineRun: PipelineRunKind): Pipeli
   };
 };
 
-export const getTaskStatus = (pipelinerun: PipelineRunKind): TaskStatus => {
+const totalPipelineRunTasks = (pipelinerun: PipelineRunKind): number => {
```

Comment:
No tests?

Reply:
Added `totalPipelineRunTasks` tests for all the test data.